packages/ts-scripts/src/cdk/components/site.ts

```diff
@@ -88,33 +90,31 @@ export const createSite = async (
   console.log('deployment', deploymentBucket.deployedBucket.bucketDomainName)
 
   const distribution = createCloudfront(stack, 'front-distro', {
-    originConfigs: [
-      {
-        s3OriginSource: {
-          s3BucketSource: bucket,
-          originAccessIdentity: cloudfrontOAI,
+    defaultBehavior: {
+      origin: new S3Origin(bucket),
+      edgeLambdas: edgeLambda && [
+        {
+          functionVersion: edgeLambda.currentVersion,
+          eventType: LambdaEdgeEventType.ORIGIN_RESPONSE,
         },
-        behaviors: [{ isDefaultBehavior: true }],
-      },
-    ],
+      ],
+    },
```

Comment:
Not sure if that problem you had with S3 will be effected by this? @willmcvay

Reply:
Erm, not sure